core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -948,6 +949,18 @@ public void testCurrentTimestamp()
         assertExpression("CURRENT_TIMESTAMP", new CurrentTime(CurrentTime.Function.TIMESTAMP));
     }
 
+    @Test
+    public void testTrim()
+    {
+        assertExpression("trim(BOTH FROM ' abc ')", new Trim(Trim.Specification.BOTH, ImmutableList.of(new StringLiteral(" abc "))));
```

Comment:
Please don't use this method. It's deprecated. Switch to `assertThat(expression...`